### DISCUSSION TITLE
Where can I see all the ParameterTypes?

### DISCUSSION THREAD

**Discussion by IsayDBS** [jspsych/jsPsych#3088] [Q&A] [answered]
I don´t find all the ParameterTypes in the documentation, do you have the documentation for this?

**Comment by jodeleeuw** [comment, marked as answer]
Hi @IsayDBS 

Here's where the parameter types are defined in the code:

https://github.com/jspsych/jsPsych/blob/8f0075b8a17bde966f2d0603b236d1cef076fa57/packages/jspsych/src/modules/plugins.ts#L18-L52
